api/views.py

```diff
@@ -282,6 +284,14 @@ class AssetSpecsViewSet(ModelViewSet):
     http_method_names = ['get', 'post', 'put']
 
 
+class GroupViewSet(ModelViewSet):
```

Comment:
Update the **GroupViewset** to format data before saving it to ensure we do not have case-insensitive duplicates as below.


<img width="1650" alt="screen shot 2018-08-01 at 13 33 53" src="https://user-images.githubusercontent.com/10899666/43516898-98219c04-958f-11e8-8adb-3ea4845b9dc9.png">

Use a similar statement to this one 👇 
https://github.com/AndelaOSP/art-backend/blob/7d703b681db684b65a8a61620a842917ed706a0e/core/models/officeblock.py#L10